pep-0683.rst

```diff
@@ -503,9 +514,10 @@ mutable object won't actually be modified.
 On the other hand, some mutable objects will never be shared between
 threads (at least not without a lock like the GIL).  In some cases it
 may be practical to make some of those immortal too.  For example,
-``sys.modules`` is a per-interpreter dict that we do not expect to ever
-get freed until the corresponding interpreter is finalized.  By making
-it immortal, we no longer incur the extra overhead during incref/decref.
+``sys.modules`` is a per-interpreter dict that we do not expect to
```

Comment:
FYI, if you like, you can use Intersphinx to actually cross-reference this in the CPython docs, i.e. 

```suggestion
:data:`python:sys.modules` is a per-interpreter dict that we do not expect to
```

Reply:
That's good to know.  Thanks.  It isn't particularly beneficial in this case but I can think of other places where it might be.